lively.ide/studio/sidebar-flap.cp.js

```diff
@@ -28,41 +28,39 @@ export class SidebarFlap extends ViewModel {
 
   async toggleSidebar () {
     const sidebarIsFadingOut = !!this.world().get(this.target);
-    this.world().withTopBarDo(async topBar => {
-      let sceneGraphPresent = false;
-      sceneGraphPresent = !!topBar.sceneGraph;
-      const targetSidebar = await topBar.openSideBar(this.target);
+    let sceneGraphPresent = false;
+    sceneGraphPresent = !!this.world().sceneGraph;
+    const targetSidebar = await this.world().openSideBar(this.target);
 
-      if (sidebarIsFadingOut) {
-        this.view.animate({
-          left: (this.target === 'scene graph') ? 0 : $world.visibleBounds().width - this.view.width,
-          duration: 300
-        });
+    if (sidebarIsFadingOut) {
+      this.view.animate({
+        left: (this.target === 'scene graph') ? 0 : $world.visibleBounds().width - this.view.width,
+        duration: 300
+      });
```

Comment:
Can we just store the world in a local var here and reference it consistently all throughout the block?